examples/flows/standard/basic-with-embedding-tool/README.md

```diff
@@ -0,0 +1,114 @@
+# Basic flow with embedding tool
+
+A basic standard flow using an Azure OpenAI connection and the builtin embedding tool to generate embeddings for input text.
```

Comment:
please follow this to make sample run in CI: https://github.com/microsoft/promptflow/blob/main/examples/CONTRIBUTING.md